librz/debug/p/debug_bochs.c

```diff
@@ -322,7 +322,7 @@ static int rz_debug_bochs_wait(RzDebug *dbg, int pid) {
 	}
 	desc->data[0] = 0;
 
-	return true;
+	return RZ_DEBUG_REASON_SIGNAL;
```

Comment:
It would be better to return RZ_DEBUG_REASON_NONE in this case instead of reporting that a signal was received